src/node_dotenv.cc

```diff
@@ -8,6 +9,17 @@ namespace node {
 using v8::NewStringType;
 using v8::String;
 
+/**
+ * The inspiration for this implementation comes from the original dotenv code,
+ * available at https://github.com/motdotla/dotenv/blob/master/lib/main.js#L12
```

Comment:
Please use permalinks instead of referring to the current revision on some branch (or just refer to the repository, I suppose).

Reply:
Done